change/office-ui-fabric-react-2020-08-18-19-00-44-details-list-focus.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

```suggestion
  "type": "minor",
```

Also mention the new prop being added to FocusTrapZone.These go into release notes.